modules/serverless-kn-container.adoc

```diff
@@ -0,0 +1,60 @@
+[id="serverless-kn-container_{context}"]
+= Knative client multi-container support
+
+You can use the `kn container add` command to print YAML container spec to standard output. This command is useful for multi-container use cases because it can be used along with other standard `kn` flags to create definitions. The `kn container add` command accepts all container-related flags that are supported for use with the `kn service create` command. The `kn container add` command can also be chained through Unix pipes (`|`) to create multiple container definitions at once.
```

Comment:
> This command is useful for multi-container use cases because it can be used along with other standard `kn` flags to create definitions. 

Nit: Would s/deployments/'use cases' work here? 

I am trying to think of a minimalist and i10n way to avoid _use_ here, since the _useful_ and _can be used_ are essential. If we switch _can be used,_ to active voice, I think the meaning starts to drift: "You can use this command along with other standard `kn` flags to create definitions for multi-container deployments." WDYT?
That said, I don't find your usage confusing.

Reply:
I am going to leave this as is for now. I think changing it from "use cases" would change the meaning too much - deployment has a whole other meaning that IMO doesn't apply here.